src/api-clients/storage-api-client.js

```diff
@@ -7,19 +7,19 @@ class StorageApiClient {
     this.httpClient = httpClient;
   }
 
-  getObjects(prefix) {
+  getCdnObjects(parentPath) {
```

Comment:
Working with "path" instead of "prefix" makes it so much more understandable now I feel.

Reply:
Ok, I see! Then I agree that this is a better approach, you're basically pushing down the CDN model as much as possible 👍